REFERENCE.md

```diff
@@ -297,19 +310,20 @@ Default value: ``undef``
 
 ##### `peers`
 
-Data type: `Any`
+Data type: `Chrony::Servers`
 
 This selects the servers to use for NTP peers (symmetric association).
-It is an array of servers.
+It can be an array of peers or a hash of peers with their respective options.
 
 Default value: `[]`
 
 ##### `servers`
 
-Data type: `Variant[Hash,Array[Stdlib::Host]]`
+Data type: `Chrony::Servers`
 
 This selects the servers to use for NTP servers.  It can be an array of servers
-or a hash of servers to their respective options.
+or a hash of servers to their respective options. If an array is used, `iburst` will be configured for each server.
+If you don't want to use `iburst`, use a hash instead.
```

Comment:
Perhaps an `@example` is useful for this.

Reply:
Will do.